### DISCUSSION TITLE
3.3.1 Released

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#2558] [General]
Announcement published here: https://www.oqtane.org/blog/!/54/oqtane-3-3-0-released

**Comment by leigh-pointer** [comment]
In my Oqtane instance in the System Update section is saying "Framework Is Already Up To Date" but its running 3.2.1 of the framework.

**Comment by sbwalker** [comment]
@leigh-pointer The 3.3.0 release has now been updated - which means the System Update should report that a new version is available.

**Comment by leigh-pointer** [comment]
@sbwalker Thanks but the 3.3.0 nuget package is not referencing the correct Project URL
![image](https://user-images.githubusercontent.com/9447612/212209489-a9a6ef7a-94a5-468f-861e-1fec46bde7df.png)

**Reply by sbwalker** [reply]
Thank you for letting me know... I will work on a resolution.

**Reply by sbwalker** [reply]
@leigh-pointer please try this again... although it is impossible to replace a Nuget package on nuget.org, I was able to replace the Oqtane.Framework.3.3.0.nupkg file on Github and point the Marketplace entry for this item to Github. This should resolve the issue.

**Reply by leigh-pointer** [reply]
@sbwalker That works, thanks.  Upgraded like a charm.

**Reply by sbwalker** [reply]
@leigh-pointer excellent! I am glad this workaround was available... in the future I need to make that the Framework nuspec is always properly updated. Do you always use the automated System Update option within the framework to upgrade to the latest version?

**Reply by leigh-pointer** [reply]
@sbwalker I used to do everything by by hand Zip unpack and the FTP to my webserver, but when you have a few instances running then the System Update option within the framework is a no brainer.

**Reply by sbwalker** [reply]
That's good to hear... the auto update framework capability was challenging to develop so I am glad it is providing value.

**Comment by sbwalker** [comment]
Note that @leigh-pointer found an issue with a primary use case in 3.3.0 so a 3.3.1 patch release was published today (https://github.com/oqtane/oqtane.framework/releases/tag/v3.3.1)
